low/low_test.go

```diff
@@ -9,10 +9,12 @@ import "testing"
 import "math/rand"
 import "time"
 
-func init() {
+func TestInit(t *testing.T) {
```

Comment:
Why?

Reply:
to use t.FatalF

Reply:
Bad practice. If it wasn't initialized test will also fail.
In you case what if user use go test -test=TestReorder ?

Reply:
use log.Fatal if you want. Or nothing.